dist/bump-release.sh

```diff
@@ -24,8 +24,8 @@ git pull --ff-only ${ORIGIN} master
 replace_version ${VERSION}
 git tag -a -m "${VERSIONTAG}" "${VERSIONTAG}"
 
-# bump ver to +git and push to origin
-replace_version "${VERSION}+git"
+# bump ver to placeholder and push to origin
+replace_version "${VERSION}+was-not-built-properly"
```

Comment:
I actually don't care that much. I want people to build with the `build` script so this will inform that to do so and thus I like this +was-not-built-properly. But if you have strong feelings for +git, I'll be happy to revert.